diffblue.yml

```diff
@@ -1,4 +1,2 @@
-branch: master
-language: java
-buildCmd: "mvn build"
-testCmd: "mvn cobertura:cobertura"
+buildCmd: "mvn test"
```

Comment:
Yes, i've got one thanks. the tag at 3.1.2 builds successfully. I think anything newer than that doesn't.